pcdet/models/dense_heads/point_head_template.py

```diff
@@ -99,7 +99,7 @@ def assign_stack_targets(self, points, gt_boxes, extend_gt_boxes=None,
                 raise NotImplementedError
 
             gt_box_of_fg_points = gt_boxes[k][box_idxs_of_pts[fg_flag]]
-            point_cls_labels_single[fg_flag] = 1 if self.num_class == 1 else gt_box_of_fg_points[:, -1].long()
+            point_cls_labels_single[fg_flag] = 1 if self.num_class == 1 else gt_box_of_fg_points[:, 7].long()
```

Comment:
Isn't `self.num_class` always a list now? If that's the case this condition will always be false. 

I guess many conditions can be automatically False because of that. Have you checked all the occurrences of `.num_class`=

Reply:
Changed it